test/fixtures/poro.rb

```diff
@@ -1,6 +1,10 @@
 class Model
   FILE_DIGEST = Digest::MD5.hexdigest(File.open(__FILE__).read)
 
+  def self.create(params)
+    @attributes = params
+  end
```

Comment:
Right, but don't you want `Model.create(params)` to return an instance of that model with those attributes?